reana/cli.py

```diff
@@ -1512,29 +1512,29 @@ def install_client():  # noqa: D301
     '--insecure-url', is_flag=True,
     help='REANA Server URL with HTTP.')
 @click.option(
-    '--namespace',
-    default='default',
-    help='Namespace of the components which configuration should be produced.')
-def setup_environment(server_hostname, insecure_url, namespace):  # noqa: D301
+    '--instance-name',
+    help='Instance name which configuration should be produced.')
```

Comment:
The help message is not clear, perhaps something like:  "REANA instance name" even though we might add a word about minikube profile name...